CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Windows, added `WindowExtWindows::set_undecorated_shadow` and `WindowBuilderExtWindows::with_undecorated_shadow` to draw the drop shadow behind a borderless window.
 - On Windows, fixed default window features (ie snap, animations, shake, etc.) when decorations are disabled.
 - On macOS, add support for two-finger touchpad magnification and rotation gestures with new events `WindowEvent::TouchpadMagnify` and `WindowEvent::TouchpadRotate`.
+- On Wayland, if not otherwise specified use upstream automatic CSD theme selection.
```

Comment:
I don't think that it matters. It was automatic for the users already? And upstream method assumes that you have portal(which I don't for example).

Reply:
Previously it defaulted to "light", in what way was it automatic? 

The new functionality upstream will use system settings to select theme. e.g. for me on gnome it will automatically use "dark" as indicated by dbus. If no dbus / dbus system preference, upstream continues to pick "light". We could add more methods of fetching system theme preference upstream, if they exist.

Reply:
It was automatic in a way that there were no explicitly stated default, so whatever winit was using for the default was up to winit. So in that regard I'm not sure that it should be in `CHANGELOG` at all, given that there's no change in user facing docs compared to previous version or breaking change, since if you were relied on winit using the light theme you were relying on implementation behavior.

We could still keep it like that, I don't really care, just don't want to have overly long changelog in the end.

Reply:
I put it in the change log because it's a change that may be interesting to winit users. For example, if downstream had been setting the theme themselves because it didn't match the system dark-preference they could read this and see if the automatic selection satisfies their use case.

Isn't this the kind of thing a changelog is for?

Reply:
I guess it's fine then.

We in alacritty has our own setting that you can control and live update via IPC commands.